macosx/Makefile

```diff
@@ -30,16 +31,17 @@ NMAP_UPDATE_BUILD_DIR = nmap-update-build
 NMAP_UPDATE_STAGING_DIR = nmap-update
 
 JHBUILD_PREFIX=$(HOME)/gtk/inst
+OPENSSL_PREFIX=$(HOME)/gtk/source/openssl-1.0.2h
```

Comment:
We shouldn't be pointing to anything in `~/gtk/source` or hard-coding any versions in the Makefile. The configure scripts should find the appropriate libraries within `~/gtk/inst` if they are told that's where to find them with the various `--with-*` options